### DISCUSSION TITLE
Improve Default Material

### DISCUSSION THREAD

**Discussion by shankarsivarajan** [bnpr/Malt#361] [Feedback & Feature Requests]
The featureless black silhouette material (or lack thereof) you get upon switching the render engine to Malt is rather uninviting. 

It would be nice if there was a default material applied to all objects that looked something more like, say, a flat gray with a black outline:

![Malt Default Material Proposal](https://user-images.githubusercontent.com/16606427/180334879-a10ba13c-0569-4205-8a50-df95668ee355.png)

**Comment by Kolupsy** [comment]
Hi there, you can configure your default material how you like and then save your file as a startup file for example. Or save a default shader as an asset and replace malts default shader with your own whenever you make a new file/project.

**Reply by shankarsivarajan** [reply]
Thanks. Yeah, I did that, and it's fine. My point is that it could be much easier to get started.
